README.md

```diff
@@ -1,76 +1,101 @@
-# HTML5 Media
+# Working with HTML5 Media
+
+## Problem Statement
+The internet is a highly interactive environment. Even blogs where text
+may be the most dominant content, audio and video elements often support,
+or even supplement, the content. In the case of 3rd party media sites such
+as YouTube, SoundCloud, and Vimeo, you are given an all-in-one line of code
+with configurable properties that creates the embeddable content for you--
+but how do you do this manually in your application? Often times, you will be working
```

Comment:
I'm confused: the text makes it seem like we're teaching them to generate embed codes? Or are we teaching them to use the embed codes? If so, that intention isn't getting across clearly.